common/os_calls.c

```diff
@@ -416,11 +416,27 @@ g_tcp_socket(void)
 
 #if defined(XRDP_ENABLE_IPV6)
     rv = (int)socket(AF_INET6, SOCK_STREAM, 0);
+    if (rv < 0)
+    {
+        log_message(LOG_LEVEL_ERROR, "g_tcp_socket: %s", strerror(errno));
```

Comment:
It turns out were are using g_get_strerror() instead on strerror(errno). I was wrong to use the later in my socket patches, I'll fix it shortly. strerror() has issues beyond portability, it it not thread safe. I expect we would switch to g_strerror() from glib (or whatever apr has), so having some uniformity in the pre-exisitng code would help.